docs/domain_model/schema-metadata.json

```diff
@@ -304,6 +307,28 @@
       ],
       "additionalProperties": false
     },
+    "publication": {
+      "type": "object",
+      "description": "A publication.",
+      "properties": {
+        "text": {
+          "type": "string",
+          "description": "The citation form of the publication."
+        },
+        "url": {
+          "description": "The URLs of the publication, preferably PIDs like DOI or ARK.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/url"
+          }
```

Comment:
What kind of items are this?

Reply:
Its part of the JSON schema definition: https://json-schema.org/specification

Reply:
These changes are the backport of https://github.com/dasch-swiss/dsp-meta-svc/pull/145 in the "old" service